Module/utc/sandswimmer.utc.json

```diff
@@ -472,11 +472,11 @@
         "__struct_id": 4,
         "Spell": {
           "type": "word",
-          "value": 90
+          "value": 799
```

Comment:
What was the spell change here and why was it made as part of the lot adjustments?

Reply:
No idea, apparently the Toolset decided that Invisibility needed to be Vampire Invisibility